src/AltinnCore/Designer/package-lock.json

```diff
@@ -25,7 +25,7 @@
     "@lerna/batch-packages": {
```

Comment:
If you have not updated package.json, you should not commit package-lock.json (since you have not changed any packages).

Also your installed node modules is downloaded with sha1, you should update your node_modules folder, ref: https://stackoverflow.com/questions/47638381/why-did-package-lock-json-change-the-integrity-hash-from-sha1-to-sha512?noredirect=1&lq=1